GoogleForCreators/web-stories-wp
# Pre-publish Checklist: Add triggers for page additions

## Context

Adding triggers to the Pre publish checklist (PPC).

## Summary

- When a second page is added the `recommended` section will become enabled.
- When a fifth page is added the `high priority` section will become enabled.

## Relevant Technical Choices

n/a

## To-do

- [x] update the tooltip once #7139 is enabled (looks like #7139 had already fixed it!)

## User-facing changes

<img src="https://user-images.githubusercontent.com/22185279/116116999-03cdba80-a679-11eb-85b5-30512a4392a8.gif" width="50%" />

## Testing Instructions

1. Create a new story
2. Click on checklist. All sections should be disabled
3. Add a new page to your story
4. The `Recommended` section should now be enabled
5. Keep adding pages until you get to 5. The `High Priority` section should be enabled

### QA






This PR can be tested by following these steps:

1. Create a new story
2. Click on checklist. All sections should be disabled
3. Add a new page to your story
4. The `Recommended` section should now be enabled
5. Keep adding pages until you get to 5. The `High Priority` section should be enabled

### UAT






## Reviews

### Does this PR have a security-related impact?

no

### Does this PR change what data or activity we track or use?

no

### Does this PR have a legal-related impact?

no

## Checklist




---



Fixes #7139
Fixes #7140
